static/css/mozsearch.css

```diff
@@ -717,6 +750,9 @@ span[data-symbols]:hover {
 span[data-symbols].hovered {
   background-color: var(--syntax-symbol-highlight);
 }
+span[data-symbols].selected {
+  outline: 1px dotted var(--syntax-symbol-selected-outline);
```

Comment:
I would suggest upping this to 3px.  Especially with the current light mode yellow highlighted symbol, it's very hard to notice this at 1px and still not much better at 2px, but at 3px I think it becomes obvious.

Reply:
Good point that it's not obvious, but I'm slightly hesitant to increase the outline width, given it hides the enclosing context.
Maybe we can use background color instead of outline?
Possibly the same color as hover (so, the clicked one remains with the highlight background color while operating on the navigation box), or a different color if it needs distinction between hover vs "no longer hover but selected" (so, the clicked one becomes "hovered" color, and it turns into different color while operating on the navigation box)?

Reply:
Ah, yeah, using the same color as the hover helps eliminate a concern about it potentially standing out against the existing highlight.  Then it basically only needs to stand out against the normal background, so the width becomes less important.  I also don't have a particularly strong opinion here and people can easily experiment with tweaking the CSS styling locally in devtools if they have opinions.

Reply:
Okay, updated the PR and dev channel to use the same styling as hovered.